### ISSUE TITLE
hide output

### ISSUE THREAD

**Issue by Hardexit** [MSEndpointMgr/IntuneWin32App#111]
Hello,

is it possible to suppress the output of the functions? Update-IntuneWin32AppPackageFile always outputs a huge text block which makes the whole thing very confusing for scripts.

**Comment by rodw-wilkins** [comment]
Did you try to set the WarningAction to SilentlyContinue?  Some of these are outputting what is considered a warning text.   Or you can also try to pipe the command to Out-Null?   Make sure to not have any -verbose parameter on the command either.

**Comment by Hardexit** [comment]
I have try it this way: https://github.com/Tealk/Intune-Win32-packaging/blob/master/functions.psm1#L170

**Comment by rodw-wilkins** [comment]
Yes, perhaps something else needs to be done to suppress the text.
